src/librustdoc/passes/mod.rs

```diff
@@ -296,18 +305,22 @@ impl<'a> fold::DocFolder for ImplStripper<'a> {
             }
             if let Some(did) = imp.for_.def_id() {
                 if did.is_local() && !imp.for_.is_generic() && !self.retained.contains(&did) {
+                    debug!("ImplStripper: impl item for stripped type; removing");
                     return None;
                 }
             }
             if let Some(did) = imp.trait_.def_id() {
                 if did.is_local() && !self.retained.contains(&did) {
+                    debug!("ImplStripper: impl item for stripped trait; removing");
                     return None;
                 }
             }
             if let Some(generics) = imp.trait_.as_ref().and_then(|t| t.generics()) {
                 for typaram in generics {
                     if let Some(did) = typaram.def_id() {
                         if did.is_local() && !self.retained.contains(&did) {
+                            debug!("ImplStripper: stripped item in trait's generics; \
+                                   removing impl");
```

Comment:
Bad indent.

Reply:
What's the proper way to indent this, then? The string is too long to fit on one line.

Reply:
The content of the string should be after the `"`:

```
"long
 string"

// and not:
"long
string"
```